### ISSUE TITLE
config_from_nc for regional case

### ISSUE THREAD

**Issue by ggebbie** [ggebbie/TMI.jl#104]
currently working for `config_from_mat`

also worth re-making NetCDF file

**Comment by ggebbie** [comment]
``` julia
julia> config2nc(TMIversion,A,γ,L,B)
ERROR: BoundsError: attempt to access 370986×1 Matrix{Float64} at index [370987]
Stacktrace:
  [1] getindex
    @ ./array.jl:924 [inlined]
  [2] #56
    @ ./none:0 [inlined]
  [3] iterate
    @ ./generator.jl:47 [inlined]
  [4] collect_to!
    @ ./array.jl:845 [inlined]
  [5] collect_to_with_first!
    @ ./array.jl:823 [inlined]
  [6] collect(itr::Base.Generator{Base.OneTo{Int64}, TMI.var"#56#57"{Matrix{Floa
t64}, Vector{CartesianIndex{3}}, Array{Float64, 3}}})
    @ Base ./array.jl:797
  [7] tracerinit(vec::Matrix{Float64}, I::Vector{CartesianIndex{3}}, wet::BitArr
ay{3})
    @ TMI ~/projects/TMI.jl/src/deprecated.jl:158
  [8] matfields2nc(TMIversion::String, γ::Grid)
    @ TMI ~/projects/TMI.jl/src/config.jl:684
  [9] config2nc(TMIversion::String, A::SparseMatrixCSC{Float64, Int64}, γ::Grid,
 L::Nothing, B::Nothing)
    @ TMI ~/projects/TMI.jl/src/config.jl:610
 [10] top-level scope
    @ REPL[15]:1
```

**Comment by ggebbie** [comment]
Mask length for `wet` doesn't match that for source `q`. Unclear how code previously worked.

**Comment by ggebbie** [comment]
Fixed in PR #105. Special case created where Izyx is only sampled at \gamma.interior points.
